packages/vscode-extension/src/extension.ts

```diff
@@ -678,7 +677,7 @@ function registerCodelensAndHoverProviders(context: vscode.ExtensionContext) {
   };
 
   const adaptiveCardCodeLensProvider = new AdaptiveCardCodeLensProvider();
-  const adaptiveCardFilePattern = `**/${AdaptiveCardsFolderName}/*.json`;
+  const adaptiveCardFilePattern = `**/*.json`;
```

Comment:
minor: it might search `node_modules` folder.

Reply:
This code lens will only be active when the user opens the file. So we do not need more restrictions on the folder's name.